### PR TITLE
chore: add Free React Bootcamp

### LINKED ISSUE #730: [Add] Free React Bootcamp
### Site URL

https://ui.dev/free-react-bootcamp

### Description of the above provided site

Discover a comprehensive 4-day React boot camp that offers a solid foundation in React development. Each day of the boot camp features an in-depth video, providing a comprehensive learning experience for mastering React's concepts and techniques.

### Category

Frontend

### Sub Category

React Js

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Anmol-Baranwal**
## Fixes Issue

closes #730 

## Changes proposed

- Add free React Bootcamp Resource Link under Frontend category

## Screenshots

image

## Note to reviewers

- I will resolve any merge conflicts that may arise after merging other pull requests from different branches.

### CODE REVIEW

File: database/frontend/react.json [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
+        "name": "React js",
+        "description": "Free online course for learning react js",
+        "url": "https://scrimba.com/learn/learnreact#",
+        "category": "frontend",
+        "subcategory": "react",
+        "language": "english"
+    },
+    {
+        "name": "Epic React",
+        "description": "Epic React is a paid comprehensive workshop series by Kent C. Dodds that equips intermediate to advanced web developers with the knowledge and skills to build robust, well-tested React applications that scale.",
```
Comment (CBID2) [inline database/frontend/react.json:12]: [Nitpick] This sounds like an interesting series, but we’re trying to only showcase free resources. Consider replacing this series with a free one.
```suggestion
        "description": "Epic React is a paid comprehensive workshop series by Kent C. Dodds that equips intermediate to advanced web developers with the knowledge and skills to build robust, well-tested React applications that scale.",

```

File: database/frontend/react.json [hunk 2]
```diff
@@ -1,10 +1,18 @@
 [
     {
-        "name":"React js",
-        "description":"Free online course for learning react js",
-        "url":"https://scrimba.com/learn/learnreact#",
-        "category":"frontend",
-        "subcategory":"react",
-        "language":"english"
+        "name": "React js",
+        "description": "Free online course for learning react js",
```
Comment (CBID2) [inline database/frontend/react.json:4]: [Typo]
```suggestion
        "description": "Free online course for learning React.js",

```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Anmol-Baranwal! :) I pointed out some proofreading errors and typos in my inline comments. Other than that, you’re off to a great start! :)

**Comment by Anmol-Baranwal** [comment]
I have changed the resource as Epic React was a paid resource, so I am resolving the conversation.

File: database/frontend/react.json [hunk 3]
```diff
@@ -1,10 +1,18 @@
...
+        "name": "React js",
+        "description": "Free online course for learning react js",
+        "url": "https://scrimba.com/learn/learnreact#",
+        "category": "frontend",
+        "subcategory": "react",
+        "language": "english"
+    },
+    {
+        "name": "Learning React Docs",
+        "description": "This React documentation provides comprehensive resources for learning React, including tutorials, guides, and API references. It covers fundamental concepts, advanced topics, and best practices, making it an essential reference for beginners and experienced React developers.",
```
Comment (CBID2) [inline database/frontend/react.json:12]: [Proofreading]
```suggestion
        "description": "This React documentation provides comprehensive resources for learning React, including tutorials, guides, and API references. It also covers fundamental concepts, advanced topics, and best practices, making it an essential reference for beginners and experienced React developers.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
You're close @Anmol-Baranwal! :) I just pointed out a proofreading suggestion in my inline comment.

File: database/frontend/react.json (hunk 3 above)
Comment (Anmol-Baranwal) [reply database/frontend/react.json]: I checked it on Grammarly, and it was correct :)

Anyway, I have made the changes you requested.

**Review by CBID2 — APPROVED** [review]
Great job @Anmol-Baranwal! 😄 LGTM! 😄

**Comment by Anmol-Baranwal** [comment]
Hi there! I noticed that there are a couple of issues that haven't been assigned to me. I'm wondering if this could be an issue when considering my participation in GSSoC'23.

**Comment by CBID2** [comment]
Which ones aren't assigned to you @Anmol-Baranwal?

**Comment by CBID2** [comment]
> > Which ones aren't assigned to you @Anmol-Baranwal?
> 
> Thanks, the issues has been assigned to me.

You're welcome @Anmol-Baranwal! :)

**Review by rupali-codes — COMMENTED** [review]
The language is by default English, so we don't have to add it here :)

**Comment by Anmol-Baranwal** [comment]
> The language is by default English, so we don't have to add it here :)

Done :)

**Review by CBID2 — DISMISSED** [review]
LGTM!

**Comment by rupali-codes** [comment]
@Anmol-Baranwal please resolve merge conflicts 😄

**Comment by Anmol-Baranwal** [comment]
I have made the suggested changes based on Christine's recommendation. As React 2025 was a paid course, I have replaced it with the official React documentation. This change has already been merged with PR #711 . Since I don't want to close the PR and issue without merging, I have thoroughly searched and confirmed that the new resource does not exist.

> New Resource: https://ui.dev/free-react-bootcamp

**Review by rupali-codes — APPROVED** [review]
great, thanks

File: database/frontend/react.json [hunk 4]
```diff
@@ -1,10 +1,17 @@
...
-        "url":"https://scrimba.com/learn/learnreact#",
-        "category":"frontend",
-        "subcategory":"react",
-        "language":"english"
+        "name": "React js",
+        "description": "Free online course for learning react js",
+        "url": "https://scrimba.com/learn/learnreact#",
+        "category": "frontend",
+        "subcategory": "react",
+        "language": "english"
```
Comment (rupali-codes) [inline database/frontend/react.json:8]:
```suggestion
```

Comment (Anmol-Baranwal) [reply database/frontend/react.json]: You can see the file here: https://github.com/rupali-codes/LinksHub/blob/aea64406f8f4a21f2478650ec12d7166699ff5d3/database/frontend/react.json

Apologies for the confusion. Despite not being the original contributor, I have made the necessary changes as required.

**Review by rupali-codes — DISMISSED** [review]
cool, thanks

File: database/frontend/react.json [hunk 5]
```diff
@@ -33,5 +33,12 @@
         "url": "https://react2025.com/",
         "category": "frontend",
         "subcategory": "react"
+    },
+    {
+        "name": "Free React Bootcamp",
+        "description": "Discover a comprehensive 4-day React boot camp that offers a solid foundation in React development. Each day of the boot camp features an in-depth video, providing a comprehensive learning experience for mastering React's concepts and techniques.",
```
Comment (CBID2) [inline database/frontend/react.json:39]: [Proofreading]
```suggestion
        "description": "This 4-day React boot camp offers a solid foundation in React development. Each day features an in-depth video, providing a comprehensive learning experience for mastering React's concepts and techniques.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
So close @Anmol-Baranwal! 😃

File: database/frontend/react.json (hunk 5 above)
Comment (Anmol-Baranwal) [reply database/frontend/react.json]: Definitely not close :)

**Review by CBID2 — APPROVED** [review]
LGTM!

**Pull request merged**